### DISCUSSION TITLE
Full version for self hosting

### DISCUSSION THREAD

**Discussion by jesperordrup** [plausible/analytics#3857] [Self-Hosted Support]
Any plans for a full version for self hosting? A paid server licens ?

**Comment by Simonmaribo** [comment]
If you self host using Coolify, you can easily set up your own instance of Plausible; https://coolify.io/docs/services/plausible-analytics

**Reply by jesperordrup** [reply]
Thank you. No doubt about that. Coolify is an awesome platform..

But my question was more directed towards license / version for selfhosting
